app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -1,161 +0,0 @@
-import { useContext } from 'react';
-import { t } from '@18f/identity-i18n';
-import { FormError } from '@18f/identity-form-steps';
-import { trackError } from '@18f/identity-analytics';
```

Comment:
I'll remove it if that's the team consensus, but I think it should stay for future use.

Reply:
I fall more on the side of removing anything that's unused and that can be brought back later, but I don't feel too strongly in this case, since it's honestly surprising this is the only error we log on the frontend, and seems like something we'd want to have easy access to.